boom/command.py

```
@@ -1067,7 +1069,9 @@ def create_entry(
              occurs while writing the entry file.
     """
     if not profile:
-        raise ValueError("Cannot create entry without OsProfile.")
+        raise ValueError(
+            "Cannot create entry without a profile (OsProfile or HostProfile)."
+        )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)